server/difido-server/src/main/java/il/co/topq/report/front/rest/ExecutionResource.java

```diff
@@ -65,6 +67,30 @@ public int post(ExecutionDetails executionDetails) {
 
 	}
 
+	@POST
+	@Path("/update")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces(MediaType.TEXT_PLAIN)
+	public String updateExecutionMetadata(ExecutionMetadataUpdateRequest updateRequest) {
```

Comment:
I think that we can pass the comment, description and other execution details using the already existing query string of the **PUT** request instead of adding another request.